### PR TITLE
When User Click Contributor's name his/her Profile will open

### PR DESCRIPTION
**Pull request by ujjwal-dahal**
Issue #68
When the User Click Contributor's Name his/her Profile will open .

I have created a Profile Link which is Empty in "events.json". Please update the Profile Link for Every Contributor.

### CODE REVIEW

**Comment by Abhilekhgautam** [comment]
@ujjwal-dahal Nice work! but there is a issue here:

* Your implementation won't work, if there are more than one contributors.

For this you might need to change the json file to sth like this:

```json
{
  "By" : ["name1", "name2"],
 "profileURL" : ["url1", "url2"]
}
```

**Comment by ujjwal-dahal** [comment]
I Have Updated Last Code Please Check Once @Abhilekhgautam

**Comment by Abhilekhgautam** [comment]
@ujjwal-dahal Resolve the conflicts, and we can move forward.

**Comment by ujjwal-dahal** [comment]
@Abhilekhgautam Check Once

File: site/js/event.js [hunk 1]
```diff
@@ -6,6 +6,18 @@ let today_date = date.getDate();
...
+fetch('/events.json')
+.then((data)=> data.json())
+.then((data)=>{
+   data.forEach(elt => {
+
+    const container = document.createElement('div');  
+    const title = document.createElement('a');
+    const status = document.createElement('p');
+    const guest = document.createElement('p');
+
```
Comment (Abhilekhgautam) [inline site/js/event.js:20]: why are we fetching the same thing twice?

**Comment by Abhilekhgautam** [comment]
@ujjwal-dahal the event page displays nothing and has an error in the console. Fix that

**Comment by ujjwal-dahal** [comment]
@Abhilekhgautam Check Again
Don't Know How Duplicate Code has came. I hope now it is fix.

File: site/js/event.js [hunk 2]
```diff
@@ -92,43 +91,45 @@ function renderEvents() {
...
-      title.href = elt.IMGURL || elt.URL;
+      if (elt.IMGURL) title.href = elt.IMGURL;
+      else title.href = elt.URL;
     }
 
+    title.innerText = elt.Title;
     title.classList.add("event-title");
-
-    if (event_status === "Completed") {
-      status.classList.add("event-status", "completed");
```
Comment (Abhilekhgautam) [inline site/js/event.js:112]: Don't remove the addition of the class "completed".

This class beautifies the text
image

File: site/js/event.js [hunk 3]
```diff
@@ -92,43 +91,45 @@ function renderEvents() {
...
-    } else if (event_status === "Registration Open") {
-      status.classList.add("event-status", "registration-open");
```
Comment (Abhilekhgautam) [inline site/js/event.js:114]: Don't remove the addition of class "registration-open"

This also beautifies the text

File: site/js/event.js [hunk 4]
```diff
@@ -92,43 +91,45 @@ function renderEvents() {
...
-    }
+    status.innerText = `Status: ${event_status}`;
```
Comment (Abhilekhgautam) [inline site/js/event.js:108]: No need to add the text: Status

**Review by Abhilekhgautam — CHANGES_REQUESTED** [review]

**Comment by ujjwal-dahal** [comment]
@Abhilekhgautam Is it Ok? Check Once

**Comment by Abhilekhgautam** [comment]
@ujjwal-dahal ! Great work.

Thank You again.

File: site/js/event.js (hunk 1 above)
Comment (SatyaRajAwasth1) [reply site/js/event.js]: I think he is creating element.

**Pull request merged**